test/client-error-stack-trace.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const { test } = require('tap')
+const { fetch } = require('..')
```

Comment:
can you test also the `index-fetch` file?

Reply:
Sure thing